SPECS/kernel/kernel.spec

```diff
@@ -28,7 +28,7 @@
 Summary:        Linux Kernel
 Name:           kernel
 Version:        5.15.137.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Please add a corresponding changelog entry in the `%changelog` section for this `-2` release increment